meta-lxatac-bsp/recipes-core/rauc/rauc_%.bbappend

```diff
@@ -1,8 +1,8 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
 
-RDEPENDS:${PN}:append = "e2fsprogs-resize2fs"
+RDEPENDS:${PN} += " e2fsprogs-resize2fs"
```

Comment:
Spaces at the beginning of a string are not required when using "+=".
In fact, they were required before when using `:append` :wink:

Reply:
\*sigh\*. Fixed